demo/src/index.css

```diff
@@ -0,0 +1,160 @@
+:root {
+  --background-color: hsl(0, 0%, 96%);
+  --editor-background: hsl(60, 100%, 100%);
+  --error-color: hsl(0, 85%, 62%);
+  --foreground-color: hsl(0, 0%, 0%);
+  --primary-color: hsl(189, 100%, 63%);
+  --shadow-color: hsla(0, 0%, 27%, 0.239);
+  --scrollbar-color: hsla(0, 0%, 47%, 0.4);
+  --warning-color: hsl(49, 100%, 40%);
+}
+
+@media (prefers-color-scheme: dark) {
+  :root {
+    --background-color: hsl(0, 0%, 23%);
+    --editor-background: hsl(0, 0%, 12%);
+    --foreground-color: hsl(0, 0%, 100%);
+    --shadow-color: hsl(0, 0%, 43%);
+  }
+}
+
+body {
+  background: var(--background-color);
+  display: flex;
+  flex-flow: column;
+  font-family: sans-serif;
+  height: 100vh;
+  margin: 0;
+}
+
+h1 {
+  margin: 0 auto 0 1rem;
+}
+
+.navbar {
+  align-items: center;
+  background: var(--primary-color);
+  box-shadow: 0px 5px 5px var(--shadow-color);
```

Comment:
```suggestion
  box-shadow: 0 5px 5px var(--shadow-color);
```

css `0` us unitless

Reply:
Sure.
https://stylelint.io/user-guide/rules/length-zero-no-unit covers it.

Which also comes as part of https://github.com/stylelint/stylelint-config-standard

Though it would be a tool most of unified doesn't use (related to https://github.com/orgs/mdx-js/discussions/2175)